cloud/google/clusteractuator.go

```diff
@@ -60,12 +70,39 @@ func NewClusterActuator(params ClusterActuatorParams) (*GCEClusterClient, error)
 		computeService:         computeService,
 		clusterClient:          params.ClusterClient,
 		gceProviderConfigCodec: codec,
+		eventRecorder:          params.EventRecorder,
 	}, nil
 }
 
 func (gce *GCEClusterClient) Reconcile(cluster *clusterv1.Cluster) error {
```

Comment:
was wondering if the actuator interface should follow more of the machine actuator approach, create/update/delete/exists.

the reason i ask is that the delete operation is part of reconciling, which is separate from the the reconcile in the actuator. so Reconcile in the actuator isn't really reconciling everything, just create/update/exist.

this was something that confused me while going through the code, as i was getting confused between controller vs actuator.

Either way, this wouldn't be changed in this PR.

Reply:
See https://github.com/kubernetes-sigs/cluster-api/pull/422